src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(({ sex, died }) => century
+    ? sex === 'm' && Math.ceil(died / 100) === century
+    : sex === 'm');
+
+  const result = men.reduce((a, b) => a + (b.died - b.born), 0);
```

Comment:
```suggestion
  const result = men.reduce((sum, person) => sum + (person.died - person.born), 0);
```

Также этот метод четвертым аргументом принимает весь массив людей, так что ты можешь делить на его длину здесь, но это не обязательно, просто как вариант